package.json

```diff
@@ -20,7 +20,8 @@
   "license": "ISC",
   "homepage": "https://github.com/leoasis/react-sound",
   "dependencies": {
-    "react": ">=0.13.3 <16.0.0",
+    "prop-types": "15.5.8",
```

Comment:
You should already be able to use `yarn` to develop locally. I'm not including a lockfile (or a shrinkwrap for that matter) since in general you don't want to lock your dependencies in a library (it makes sense in an app though)